test/verify/check-connection

```diff
@@ -347,6 +347,7 @@ class TestConnection(testlib.MachineCase):
 
         self.allow_journal_messages(".*cockpit-ws.*dumped core.*")
         self.allow_journal_messages(".*Error creating thread: Resource temporarily unavailable.*")
+        self.allow_journal_messages(".*Failed to check for ssh: Failed to fork.*")
```

Comment:
Wha, this happens?

Reply:
Yep -- this is testHttpsInstanceDoS, which severely restricts the resource limits for cockpit's cgroup, to check the behaviour under that circumstance. Running out of the number of processes is the very point of this.